account-kit/core/src/types.ts

```diff
@@ -62,23 +62,68 @@ export type AlchemyAccountsConfig = {
     sessionLength: number;
   };
 };
-
 // [!region CreateCorConfigProps]
-export type Connection = {
+export type ViemConnection = {
```

Comment:
we can't rename this type, that would make it a breaking change since it's exposed publicly

Reply:
The exposed is still Connection, it just that Connection is built with Eth and Solana

Reply:
Okay, removed